script/upload.py

```diff
@@ -235,6 +238,20 @@ def upload_io_to_github(github, release, name, io, content_type):
       params=params, headers=headers, data=io, verify=False)
 
 
+def upload_sha256_checksum(version, file_path):
+  bucket, access_key, secret_key = s3_config()
+  checksum_path = '{}.sha256sum'.format(file_path)
+  sha256 = hashlib.sha256()
+  with open(file_path, 'rb') as f:
+    sha256.update(f.read())
+
+  filename = os.path.basename(file_path)
+  with open(checksum_path, 'w') as checksum:
+    checksum.write('{}  {}'.format(sha256.hexdigest(), filename))
```

Comment:
This is incorrect. According to [the man page for sha256sum](http://linux.die.net/man/1/sha256sum), it needs to be

```python
checksum.write('{} *{}'.format(sha256.hexdigest(), filename))
```

(Note the asterisk)

This is causing my checksum verifications to not work (because it's reading the zips in text mode, not binary mode).

A PR to fix this is coming shortly.